src/components/Dialogs/LanguagesModal.tsx

```diff
@@ -10,16 +10,23 @@ import { useSettingsQuery } from '@/core/react-query/settings/queries';
 import useEventCallback from '@/hooks/useEventCallback';
 
 type Props = {
-  type: 'Series' | 'Episode' | null;
+  type: 'Series' | 'Episode' | 'Description' | null;
   onClose: () => void;
 };
 
 function LanguagesModal({ onClose, type }: Props) {
   const settings = useSettingsQuery().data;
   const LanguagePreference = useMemo(
-    () => (type === 'Episode'
-      ? settings.EpisodeLanguagePreference
-      : settings.LanguagePreference),
+    () => {
```

Comment:
these switches exist because I couldn't use nested ternaries. Feel free to keep or remove them.